lib/api_me/sorting.rb

```diff
@@ -26,20 +31,22 @@ def sort_meta
 
     protected
 
-    def sort
-      unless sort_criteria === ""
-        sort_p = sort_criteria
+    def sort(criteria = default_sort_criteria)
+      unless sort_association == ""
+        criteria_class = criteria.camelize.constantize
```

Comment:
It is, unless I'm missing something...

Pulled it out and ran the sorting again and unless `criteria_class` is used for the sort method it throws an error because `criteria` itself is just a String (according to the error thrown). So removing the '_id' part of the string and running `criteria.camelize.constantize` fixes that issue.